src/image_reader/free_functions.rs

```diff
@@ -148,6 +148,7 @@ static MAGIC_BYTES: [(&[u8], ImageFormat); 23] = [
     (b"\0\0\0\x1cftypavif", ImageFormat::Avif),
     (&[0x76, 0x2f, 0x31, 0x01], ImageFormat::OpenExr), // = &exr::meta::magic_number::BYTES
     (b"qoif", ImageFormat::Qoi),
+    (&[0x0A], ImageFormat::Pcx),
```

Comment:
Only the first byte is truly a magic byte. Some decoders take the first four bytes, but this can lead to false negatives.

Reply:
Great self-review. I think we want to weigh the accuracy of magic bytes such that they don't preclude alternatives. For that goal aligning with imagemagick as [they did here](https://github.com/JuliaIO/FileIO.jl/pull/114) is probably a good default. The only better alternative I could see is the functional approach of letting some patterns filter by evaluated code.

Reply:
Sure thing; I've just pushed https://github.com/image-rs/image/pull/2364/commits/f163f1449251f0049373b52b05c07974ac5ee188 which changes to checking for those two byte sequences.